x/vesting/spec/01_concepts.md

```diff
@@ -14,21 +14,28 @@ The duration until which the first tokens are vested is called the `cliff`.
 
 ## Lockup
 
-The lockup describes the schedule by which tokens are converted from a  `locked` to an `unlocked` state.
-As long as all tokens are locked, the account cannot perform any Ethereum transactions using the `x/evm` module.
+The lockup describes the schedule by which tokens are converted from a `locked` to an `unlocked` state.
+As long as all tokens are locked, the account cannot perform any Ethereum transactions
+that spend EVMOS using the `x/evm` module.
+However, the account can perform Ethereum transactions that don't spend EVMOS tokens.
 Additionally, locked tokens cannot be transferred to other accounts.
 In the case in which tokens are both locked and vested at the same time,
 it is possible to delegate them to validators, but not transfer them to other accounts.
 
 The following table summarizes the actions that are allowed for tokens
 that are subject to the combination of vesting and lockup:
 
-| Token Status            | Transfer | Delegate | Vote | Eth Txs |
-| ----------------------- | :------: | :------: | :--: | :-----: |
-| `locked` & `unvested`   |    ❌    |    ❌    |  ❌  |   ❌    |
-| `locked` & `vested`     |    ❌    |    ✅    |  ✅  |   ❌    |
-| `unlocked` & `unvested` |    ❌    |    ❌    |  ❌  |   ❌    |
-| `unlocked` & `vested`   |    ✅    |    ✅    |  ✅  |   ✅    |
+| Token Status            | Transfer | Delegate | Vote | Eth Txs that spend EVMOS\*\* | Eth Txs that don't spend EVMOS (amount = 0)\*\* |
+| ----------------------- | :------: | :------: | :--: | :--------------------------: | :---------------------------------------------: |
+| `locked` & `unvested`   |    ❌    |    ❌    |  ❌  |              ❌              |                       ✅                        |
+| `locked` & `vested`     |    ❌    |    ✅    |  ✅  |              ❌              |                       ✅                        |
+| `unlocked` & `unvested` |    ❌    |    ❌    |  ❌  |              ❌              |                       ✅                        |
```

Comment:
@danburck do you know why is it that `unlocked` accounts can not be transferred?

Reply:
nvm I got this